akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -30,6 +30,10 @@ final class Sink[-In, +Mat](private[stream] override val module: Module)
 
   override val shape: SinkShape[In] = module.shape.asInstanceOf[SinkShape[In]]
 
+  def contramap[In2](f: In2 ⇒ In): Sink[In2, Mat] = Flow.fromFunction(f).toMat(this) { (_, m) ⇒ m }
```

Comment:
I'm on the fence for this one. It is a slippery slope to add N contra methods. More choice to do the same thing (vs prepend) is not automatically better. Thoughts?